modules/serverless-ossm-installing-and-configuring-openshift-service-mesh.adoc

```diff
@@ -139,6 +139,11 @@ spec:
 ----
 <1> Name of the secret containing the wildcard certificate.
 <2> The `knative-local-gateway` serves HTTPS traffic and expects all clients to send requests using mTLS. This means that only traffic coming from within {SMProductShortName} is possible. Workloads from outside the {SMProductShortName} must use the external domain via OpenShift Routing.
++
+[NOTE]
+====
+You must configure the wildcard certificate to match the domain of your {ocp-product-title} cluster. For example, if your {ocp-product-title} console address is `https://console-openshift-console.apps.openshift.example.com`, you must configure the wildcard certificate so that the domain is `*.apps.openshift.example.com`.
```

Comment:
@rh-max Thanks for this. We probably still want to put back the note that we removed:
```
For more information about configuring wildcard certificates, see the following topic about _Creating a certificate to encrypt incoming external traffic_.
 ...
 include::modules/serverless-ossm-external-certs.adoc[leveloffset=+1]
```
And also put back the file serverless-ossm-external-certs.adoc which you're removing in this PR.
Having this example is quite important. Generating the certificates needs a lot of specific parameters so it's hart to figure it out without any example.

Reply:
@mgencur Thanks. I have removed this particular instance of the note, because I re-added the note in the "Prerequisites" section at the top of the page (and it refers to the "Creating a certificate to encrypt incoming external traffic" section, which follows right after).